greenpeace/planet4-master-theme
# PLANET-7395: Move Deep Dive block pattern into master theme

Ref: https://jira.greenpeace.org/browse/PLANET-7395

Related PR: [..pending]

# Description
Move Deep Dive block into the theme

## Testing
- Create a new page
- Edit that page
- Click on "+" and click on "Patterns" tab
- The Deep Dive block should be visible on that list
